monitor_and_download_tasks.py

```diff
@@ -0,0 +1,37 @@
+"""Monitor the tasks stored in the .json file logged with the API."""
+import os
+import json
+
+import inductiva
+
+PATH_TO_JSON = "/Path/to/task_metadata.json"
+DOWNLOAD_TASKS = True
+
+
+def main():
+    json_lines = []
+    with open(PATH_TO_JSON, "r", encoding="utf-8") as json_file:
```

Comment:
Reading each line separately doesn't seem consistent. Shouldn't we be able to load the json_file into a single structure immediately?